db/018-refactor-sessions.sql

```diff
@@ -0,0 +1,67 @@
+DO $migration$
+    DECLARE migrationId INT;
+    DECLARE lastSuccessfulMigrationId INT;
+BEGIN
+    -- The migration ID of this file
+    migrationId := 18;
+
+    -- Get the migration ID of the last file to be successfully run
+    SELECT MAX(id) INTO lastSuccessfulMigrationId
+    FROM migrations;
+
+    -- Only execute this script if its migration ID is next after the last successful migration ID
+    IF migrationId - lastSuccessfulMigrationId = 1 THEN
+        ALTER TABLE sessions
+            DROP still_counter,
+            DROP od_flag,
+            DROP duration,
+            ALTER start_time TYPE timestamptz,
+            ALTER start_time SET DEFAULT now(),
+            ALTER start_time SET NOT NULL;
+
+        ALTER TABLE sessions ADD id UUID NULL;
+        UPDATE sessions SET id = CAST(LPAD(TO_HEX(sessionid), 32, '0') AS UUID);
+        ALTER TABLE sessions 
+            DROP sessionid,
+            ALTER id SET NOT NULL,
+            ALTER id SET DEFAULT gen_random_uuid(),
+            ADD CONSTRAINT sessions_pkey PRIMARY KEY (id);
+
+        ALTER TABLE sessions RENAME start_time TO created_at;
+        ALTER TABLE sessions RENAME incidenttype TO incident_type;
+        ALTER TABLE sessions RENAME phonenumber to phone_number;
+        ALTER TABLE sessions drop column end_time;
+        ALTER TABLE sessions ADD COLUMN updated_at timestamptz DEFAULT now() NOT NULL;
```

Comment:
good question, I actually don't have a good reason. Do you have a suggestion?

Reply:
How about using `end_time`?
There are rows in the DB that have `end_time is null`, but they are all ones where `od_flag = 0`, which will be deleted earlier in this file. So that should give us a better idea of when it was last updated, don't you think?

```
backend-replacement=> select count(*) from sessions where end_time is null and od_flag=1;
 count 
-------
     0
(1 row)
```